gatsby-node.js

```diff
@@ -51,7 +51,7 @@ exports.onPostBuild = async function ({ graphql, reporter }, config) {
       await index.waitForPendingUpdate(enqueuedUpdate.updateId)
       const res = await index.getUpdateStatus(enqueuedUpdate.updateId)
       if (res.status === 'failed') {
-        throw getErrorMsg(res.message)
+        throw getErrorMsg(res.error.message)
```

Comment:
```suggestion
        throw getErrorMsg(`${res.error.message}: ${res.error.code}`)
```

It could be cool for the user to be able to search in the documentation via the error code.

Reply:
Will be done in an other PR (https://github.com/meilisearch/gatsby-plugin-meilisearch/issues/77), thanks for the suggestion ! ❤️